docs/contributing/coding-guidelines/ros-nodes/parameters.md

```diff
@@ -99,16 +99,17 @@ Note: `/**` is used instead of the explicit node namespace, this allows the para
           "type": "INSERT_TYPE",
           "description": "INSERT_DESCRIPTION",
           "default": "INSERT_DEFAULT",
```

Comment:
Do we keep the quotes for the default value? For a string or a boolean for example it takes quotes, but not for integers.

Reply:
What I noticed is that it populates the parameter file correctly if the "default" value is within quotes. E.g., it'll keep `10.0` and not populate `10`. The validation still works in VSCode.

Reply:
Ah, I see. `default` is an annotation, which "_aren’t strictly used for validation, but are used to describe parts of a
schema_". And "_default is typically used to express that if a value is missing, then the
value is semantically the same as if the value was present with the default value_". So having quotes makes perfect sense.